packages/pwa-buildpack/lib/Utilities/addImgOptMiddleware.js

```diff
@@ -92,11 +103,21 @@ https://github.com/nodejs/node-gyp#installation`
             //   https://github.com/magento-research/express-sharp/blob/develop/lib/transform.js#L23
             //   https://sharp.pixelplumbing.com/en/stable/api-resize/
             if (height) {
-                rewrittenUrl += `/${height}`;
-                params.set('crop', true);
+                params.set('height', height);
+                if (!crop) {
```

Comment:
`crop = true` as replacement for 107-109 comes to mind

Reply:
I've updated the logic to handle this in an easier to understand way:
```
            if (crop) {
                params.set('crop', crop);
            } else if (width && height) {
                // If we received height and width we should force crop since our
                // implementation of express sharp defaults fit to "outside" if crop
                // is falsy. `outside` sizes the image, retaining the aspect ratio
                // but may fall "outside" the desired height or width. `cover`
                // retains the aspect ratio like `outside` but clips to fit desired
                // height and width.
                //   https://github.com/magento-research/express-sharp/blob/develop/lib/transform.js#L23
                //   https://sharp.pixelplumbing.com/en/stable/api-resize/
                params.set('crop', true);
            }
```